cwf/gateway/services/uesim/servicers/uesim_hssless.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	cwfprotos "magma/cwf/cloud/go/protos"
+	"magma/feg/gateway/services/aaa/session_manager"
+	lte_protos "magma/lte/cloud/go/protos"
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/lib/go/protos"
+	"strings"
+
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
+)
+
+const (
+	imsiPrefix    = "IMSI"
+	defaultUeIp   = "192.168.88.88"
```

Comment:
Don't use defaults. Pass those parameters in the protos. You can create a new message on each proto with `HssLess` sufix. 

for Example 
```
message AuthenticateRequest {
    string imsi = 1;
    string calledStationID = 2;
	AuthenticateRequestHssLess hssLessConfig = 3;
}

message AuthenticateRequestHssLess {
	string ip = 1;
	string apn = 2; 
}

```
Same with the disconnect or any other call that requires any extra parameter

Reply:
Incorporated the review comment by adding  AuthenticateRequestHssLess as separate type and encapsulating it inside AuthenticateRequest.